packages/SwingSet/src/kernel/kernel.js

```diff
@@ -756,12 +797,21 @@ export default function buildKernel(kernelEndowments) {
       // now the vatManager is attached and ready for transcript replay
     }
 
+    function terminateVat(vatID) {
+      const vatKeeper = kernelKeeper.allocateVatKeeperIfNeeded(vatID);
+      vatKeeper.markAsDead();
+      removeVatManager(vatID).then(
+        () => console.debug(`terminated vat ${vatID}`),
```

Comment:
I'm ok with this `console.debug` for now, but in the longer run I'm aiming to only print things to the console if something went wrong, and I think this is in the works-as-expected category.

Reply:
Oh, I think that should have been a `kdebug` call.